client/signup/config/flows.js

```diff
@@ -219,6 +219,11 @@ function filterFlowName( flowName ) {
 			return ( 'blog' === flowName ) ? 'blog-altthemes' : 'website-altthemes';
 		}
 	}
+
+	if ( 'skipForFree' === abtest( 'freePlansDefault' ) ) {
```

Comment:
Currently, users in the `notTested` and `original` variations of the `headstart` and `altThemes` tests above will be assigned to either variation of `freePlansDefault`. This affects the results of the other tests: for instance, if the `skipForFree` variation of this test performs well or poorly, it will affect the `original` variation of the two other tests, and the results could be misleading.

I think we should set this test up such that only users from the `notTested` variations above are assigned (cc @sirbrillig or @kwight to confirm that this was their intention with that variation) so that the definition of `original` is still whatever the default flow is.